trunner/runners/common.py

```diff
@@ -265,7 +122,10 @@ def open(self):
             raise
 
         try:
-            self.plo = pexpect.fdpexpect.fdspawn(self.serial, encoding='ascii', timeout=8)
+            self.plo = pexpect.fdpexpect.fdspawn(
```

Comment:
No need to format it like this, if u want to keep consistent style just use `black` or other formatter

Reply:
Changed to one line